fdbserver/worker.actor.cpp

```diff
@@ -1332,14 +1333,18 @@ struct TrackRunningStorage {
 	UID self;
 	KeyValueStoreType storeType;
 	std::set<std::pair<UID, KeyValueStoreType>>* runningStorages;
+	static int startedCount; // how many times storage server has started on this worker
```

Comment:
Static class members are shared across all Workers of all virtual processes in simulation.  If you want to track something per Worker, it should probably be done in the Worker role.

Reply:
fixed. thanks